### ISSUE TITLE
<Ctrl+.>doesn't work

### ISSUE THREAD

**Issue by dapanghu** [brookhong/Surfingkeys#1003]
## Error details

SurfingKeys: 0.9.45

Browser: Chrome Version 75.0.3770.100 (Official Build) (64-bit)

OS: OSX

## Context

<Ctrl+.> won't show results of next page. When I press these 2 keys, nothing happened. But  <Ctrl+,> works well.

**Comment by type-writer** [comment]
我也有同样的问题,不过和你相反,是<ctrl-,>无效,而<ctrl-.>好用

**Comment by hoohyaIII** [comment]
I have the same question . Ctrl+. do not work , but Ctrl+ , works. I think maybe it has some relativity to my Ji Dian wubi 98 Input Method Editor.

**Comment by RuichenANCC** [comment]
@hoohyaIII Yeah Bro u r correct. It is the problem with '.'. When I exchange it with <Ctrl-/> everything is now okay.
I am using MacOS 15. It might be a problem with MacOS.
Thank you Bro!

兄弟你说得对，就是'.'的问题。换成/一下子就好了。研究了几个星期了，才想起来到issue看一看一下子就解决了。感谢!

**Comment by yangguanghaitan** [comment]
@hoohyaIII perfect
